dashboard/lib/tasks/seed.rake

```diff
@@ -9,8 +9,7 @@ namespace :seed do
 
   STANFORD_HINTS_FILE = 'config/stanford-hints-bestPath1.tsv'
   STANFORD_HINTS_IMPORTED = 'config/scripts/.hints_imported'
-  file STANFORD_HINTS_IMPORTED => STANFORD_HINTS_FILE do
-    require_relative '../../config/environment'
+  file STANFORD_HINTS_IMPORTED => [STANFORD_HINTS_FILE, :environment] do
```

Comment:
yes- if I recall running `rake db:setup_or_migrate seed:all` with the changes in this PR threw an exception on this manual 'require', which I suspected was because it was trying to re-load the `environment` after having loaded it once already. In any case, moving this manual require into the task dependency brings this task in line with the rest of the task definitions and fixed the exception.